src/lib/firebase.ts

```
@@ -59,20 +69,31 @@ const envConfig: FirebaseRuntimeConfig = {
+      ) as Partial<FirebaseRuntimeConfig> | undefined)
     : undefined;
 
 const firebaseConfig: FirebaseRuntimeConfig = {
+  ...(FALLBACK_FIREBASE_CONFIG as FirebaseRuntimeConfig),
   ...envConfig,
   ...(injectedConfig ?? {}),
 };
+
+for (const [key, fallbackValue] of Object.entries(FALLBACK_FIREBASE_CONFIG)) {
+  const currentValue = (firebaseConfig as any)[key];
+  if (isMissing(currentValue) && !isMissing(fallbackValue)) {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)